task2/app.js

```diff
@@ -0,0 +1,127 @@
+function createUser() {
+  window.location.href = "./createUser.html";
+}
+function logout() {
+  window.location.href = "./index.html";
+  localStorage.clear();
+}
+
+const container = document.getElementById("container");
+const url = "https://reqres.in/api/users";
+let usersloggedin = new Array();
+
+try {
+  fetch(url)
+    .then((res) => res.json())
+    .then((data) => {
+      let persons = data;
```

Comment:
1. Try to avoid creating new variables wherever possible to prevent unnecessary memory allocation.
2. Destructure the values for more readability.
3. Also, either attach the 'return' keyword with the values after curly braces or simply remove the curly braces and move the values to return in the same line.
4. 
Example:

```
try {
    fetch(url)
      .then((res) => res.json())
      .then(({ data }) => data.map(({ first_name, email, avatar }) => {
          return usersloggedin.push({
            name: first_name,
            email: email,
            img: avatar,
          });
        });
}
```

@shubhamshukla91